wallet/src/repository/wallets/mod.rs

```diff
@@ -73,19 +66,22 @@ impl<T: Database> Wallets<T> {
         let types::CreateWalletData {
             id,
             name,
-            caption,
+            description,
             iv,
             salt,
             account,
         } = wallet_data;
         let mut wbatch = wallet_db.batch();
 
-        if let Some(name) = name {
+        // We first write name and description into private wallet DB
+        if let Some(name) = name.clone() {
             wbatch.put(keys::wallet_name(), name)?;
         }
```

Comment:
Maybe something like this could work?
```suggestion
        if let &Some(name) = name.as_ref() {
            wbatch.put(keys::wallet_name(), name.clone())?;
        }
```

Reply:
Or maybe we can do `batch.put` directly here as well and avoid having this `if` twice?

Reply:
I liked the second option and already pushed it